go/consensus/tendermint/apps/registry/transactions.go

```diff
@@ -663,7 +672,7 @@ func (app *registryApplication) registerRuntime( // nolint: gocyclo
 
 		switch rtToCheck.GovernanceModel {
```

Comment:
This could use the `rtToCheck.StakingAddress()` now?

Reply:
Fixed, but I've still left a switch in the error case, so we can have more relevant error messages.